bftengine/src/preprocessor/PreProcessor.hpp

```diff
@@ -175,6 +176,39 @@ class PreProcessor {
   concordUtil::Timers::Handle requestsStatusCheckTimer_;
   const uint64_t preExecReqStatusCheckPeriodMilli_;
   concordUtil::Timers &timers_;
+
+  // 5 Minutes
+  static constexpr int64_t MAX_VALUE_MICROSECONDS = 1000 * 1000 * 60 * 5l;
```

Comment:
Put just a number. 
Add a comment with a value in seconds.

Reply:
Done.